src/Forms/FieldList.php

```diff
@@ -244,10 +229,8 @@ public function addFieldsToTab($tabName, $fields, $insertBefore = null)
      *
```

Comment:
I can't target it directly but the line above this is confusing. This isn't strictly part of what you're doing so I won't block merging if you choose not to do this:

```diff
- Remove the given field from the given tab in the field.
+ Remove the given field from the given tab in the fieldlist.
```

Reply:
Done